### PR TITLE
Unix locale installation

### PR DESCRIPTION
**Pull request by malahx**
Hello, I also experienced this issue on the package build for Arch Linux #115
If needs: https://aur.archlinux.org/packages/purple-discord-git/

### CODE REVIEW

**Comment by Penaz91** [comment]
I'm going to test this immediately, thank you.

File: Makefile [hunk 1]
```diff
@@ -126,6 +126,7 @@ install-icons: discord16.png discord22.png discord48.png
 	install -m $(FILE_PERM) -p discord48.png $(DISCORD_ICONS_DEST)/48/discord.png
 
 install-locales: $(LOCALES)
+	mkdir -m $(DIR_PERM) -p $(LOCALEDIR)/es/LC_MESSAGES
```
Comment (EionRobb) [inline Makefile]: Isn't there a flag for `install` that will auto-create the folders if they don't exist?

**Comment by EionRobb** [comment]
You're awesome, thanks for this :)

File: Makefile (hunk 1 above)
Comment (malahx) [inline Makefile]: Yes, install -D ...
Corrected ;)

**Comment by Penaz91** [comment]
Confirmed working, thank you

**Pull request merged**